tests/test_e2e.py

```diff
@@ -1036,5 +1040,54 @@ def test_acquire_token_silent_with_an_empty_cache_should_return_none(self):
         #       If this test case passes without exception,
         #       it means MSAL Python is not affected by that.
 
+
+@unittest.skipUnless(broker_available, "AT POP feature is only supported by using broker")
+class PopTestCase(LabBasedTestCase):
+    def test_at_pop_should_contain_pop_scheme_content(self):
+        auth_scheme = PopAuthScheme(
+            http_method="GET",
+            url="https://www.Contoso.com/Path1/Path2?queryParam1=a&queryParam2=b",
+            nonce="placeholder",
+            )
+        config = self.get_lab_user(usertype="cloud")
+        config["password"] = self.get_lab_user_secret(config["lab_name"])
+        result = self._test_username_password(auth_scheme=auth_scheme, **config)
+        self.assertEqual(result["token_type"], "pop")
+        payload = json.loads(decode_part(result["access_token"].split(".")[1]))
```

Comment:
Consider asserting that an RT is not in the cache. WAM should not give back an RT.

Reply:
I guess what you really want to assert is that the `token_source` should be `broker`. I'm adding that assertion in my next commit.
We do not need to assert "WAM should not give back an RT". That is a WAM behavior that MSAL does not control, and shall not make an assertion on.